Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/TintedTransparent_StandardLighting.shader.template

```diff
@@ -0,0 +1,52 @@
+{
+    "Source" : "INSERT_AZSL_PATH_HERE",
+
+    "DepthStencilState" :
+    {
+        "Depth" :
+        {
+            "Enable" : true,
+            "CompareFunc" : "GreaterEqual",
+            "writeMask": "Zero"
+        },
+        "Stencil" :
+        {
+            "Enable" : true,
+            "ReadMask" : "0x00",
+            "WriteMask" : "0xFF",
+            "FrontFace" :
+            {
+                "Func" : "Always",
+                "DepthFailOp" : "Keep",
+                "FailOp" : "Keep",
+                "PassOp" : "Replace"
+            }
+        }
+    },
+    
+    "GlobalTargetBlendState" : 
+    {
+        "Enable" : true,
+        "BlendSource" : "One",
+        "BlendDest" : "ColorSource1",
+        "BlendOp" : "Add"
+    },
+
+    "ProgramSettings":
+    {
+      "EntryPoints":
+      [
+        {
+          "name": "VertexShader",
+          "type": "Vertex"
+        },
+        {
+          "name": "PixelShader",
+          "type": "Fragment"
+        }
+      ]
+    },
+
+    "DrawList" : "transparent"
```

Comment:
Note to reviewers: DrawList is different between the **[tinted]transparent** and **opaque** .shader.template files